infra/docker-compose.yaml

```diff
@@ -20,6 +20,6 @@ services:
       POSTGRES_USER: app_user
       POSTGRES_PASSWORD: password
 # Uncomment this if you want to persist the data. Create your boostrap SQL file in the project root
-#    volumes:
-#      - "./pgdata:/var/lib/postgresql/data"
-#      - "./init.sql:/docker-entrypoint-initdb.d/init.sql"
+    volumes:
+      # - "./pgdata:/var/lib/postgresql/data"
```

Comment:
Guessing we're keeping this for local testing?

Reply:
Yeah thats the only use for that, we could probably remove it later as I haven't used it myself.